internal/injector/builtin/testdata/client/vault.go.snap

```diff
@@ -0,0 +1,50 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+package main
+
+import (
+	"net/http"
+
+	"github.com/hashicorp/vault/api"
+//line <generated>:1
+	__orchestrion_vaulttrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/hashicorp/vault/api"
+)
+
+//line samples/client/vault.go:14
+func vaultClient() {
+	c, err := api.NewClient(&
+//line <generated>:1
+//line samples/client/vault.go:15
```

Comment:
Okay -- this is weird (but also this is not your PR's fault so don't go try to fix that).